man/bootctl.xml

```diff
@@ -258,23 +258,60 @@
       </varlistentry>
 
       <varlistentry>
-        <term><option>--make-machine-id-directory=yes|no</option></term>
-        <listitem><para>Control creation and deletion of the top-level machine ID directory on the file
-        system containing boot loader entries (i.e. beneath the file system returned by the
-        <option>--print-boot-path</option> option, see above) during <option>install</option> and
-        <option>remove</option>, respectively. Defaults to <literal>no</literal>. See
+        <term><option>--make-entry-directory=yes|no</option></term>
+        <listitem><para>Controls creation and deletion of the <ulink
+        url="https://systemd.io/BOOT_LOADER_SPECIFICATION">Boot Loader Specification</ulink> Type #1 entry
+        directory on the file system containing resources such as kernel images and initial RAM disk images
+        during <option>install</option> and <option>remove</option>, respectively. The directory is named
+        after the entry token, as specified with <option>--entry-token=</option> parameter described below,
+        and is placed immediately below the <varname>$BOOT</varname> root directory (i.e. beneath the file
+        system returned by the <option>--print-boot-path</option> option, see above). Defaults to
+        <literal>no</literal>.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--entry-token=</option></term>
+
+        <listitem><para>Controls how to name and identify boot loader entries for this OS
+        installation. Accepted during <option>install</option>, and takes one of <literal>auto</literal>,
+        <literal>machine-id</literal>, <literal>os-id</literal>, <literal>os-image-id</literal> or an
+        arbitrary string prefixed by <literal>explicit:</literal> as argument.</para>
```

Comment:
it's more about keeping the door open here, so that you can do file:/foo/bar/baz.txt later on if you like. i mean, there are so many ways to pick an id here... could also be some key derived from the tpm, so that oyu bind the entry to the tpm. and in that case you might want to write tpm2:/dev/tpmrm0 or so... you never know what might make sense to add here one day...

Reply:
will rename to "literal"